### ISSUE TITLE
Test RPM Build

### ISSUE THREAD

**Issue by jdinan** [Sandia-OpenSHMEM/SOS#645]
Here's how I do it.  Note that this has to be done from the top-level directory of the SOS repository for the distribution tarball to come out correctly:
```
$ ./configure --with-ofi=$HOME/opt/libfabric --enable-pmi-simple
$ make dist
$ rpmbuild -ta ./sos-1.4.1.tar.gz --define "configargs --enable-remote-virtual-addressing --enable-pmi-simple --with-ofi=$HOME/opt/libfabric"
```

**Comment by wrrobin** [comment]
@jdinan `make dist` command is not generating the tarball correctly. The tarball is only about 16 bytes. Is there anything I am missing here? I am looking at the Makefile. The command did not generate any error.

**Comment by wrrobin** [comment]
The issue regarding `make dist` is related to the tarball creation command. If I use `tar -cjf` then it works fine.

**Comment by wrrobin** [comment]
Should we then change the tarball creation to `tar` and keep that?

**Comment by wrrobin** [comment]
@jdinan, The rpmbuild command generates this error. 
```
RPM build errors:
    Installed (but unpackaged) file(s) found:
   /usr/lib64/pkgconfig/sos.pc
```
Is it ok to add `%define _unpackaged_files_terminate_build 0` in the spec file to avoid this?

**Comment by jdinan** [comment]
Could you try adding `%{_libdir}/pkgconfig/*.pc` to the `%files` and `%files devel` sections in `sandia-openshmem.spec.in`?

**Comment by wrrobin** [comment]
Thanks @jdinan. This worked. `rpmbuild` generated the following files. 
```
total 0
drwxr-xr-x. 3 rahmanmd users 26 Mar 28 12:59 BUILD
drwxr-xr-x. 2 rahmanmd users  6 Mar 28 13:01 BUILDROOT
drwxr-xr-x. 3 rahmanmd users 20 Mar 28 13:01 RPMS
drwxr-xr-x. 2 rahmanmd users  6 Mar 22 10:33 SOURCES
drwxr-xr-x. 2 rahmanmd users  6 Mar 28 13:01 SPECS
drwxr-xr-x. 2 rahmanmd users 47 Mar 28 13:01 SRPMS
```
Let me know if this looks ok?

**Comment by wrrobin** [comment]
Checked and closed by #656
